src/components/Home.js

```diff
@@ -2,7 +2,9 @@ import React from "react";
 import ContructionNoticeCreate from "./ContructionNoticeCreate";
 
 const Home = () => {
-    return <div><ContructionNoticeCreate /></div>
+    return <div>
+        <ContructionNoticeCreate />
+    </div>
```

Comment:
Can we also fix these please? Maybe you can merge main first as I've already merged the PR for this type to the main.

Reply:
Fixed the conflict. Should be good for main.